pkg/cli/admin/release/extract_tools.go

```diff
@@ -333,7 +384,7 @@ func (o *ExtractOptions) extractCommand(command string) error {
 			target.Mapping.To = filepath.Join(dir, target.Mapping.Name)
 		} else {
 			target.Mapping.To = filepath.Join(dir, target.Command)
-			target.Mapping.Name = fmt.Sprintf("%s-%s", target.OS, target.Command)
+			target.Mapping.Name = fmt.Sprintf("%s-%s-%s", target.OS, target.Arch, target.Command)
```

Comment:
Did this change the output of extract?  That's a breaking change (we can't change filenames on disk that might be used as part of automation).

If so, this needs to be reverted and replaced with a change that only changes the names on the net new components.

Reply:
No, that bit is used only for internal tracking of what has been extracted.